client/reader/recent/style.scss

```diff
@@ -178,18 +194,22 @@
 	}
 
 	&__post-column {
-		display: none;
 		@extend %column-shared;
+		display: none;
+		max-height: calc( 100vh - var( --masterbar-height ) - var( --content-padding-top ) - var( --content-padding-bottom ) );
+		height: fit-content;
+		overflow-y: auto;
 
 		&.overlay {
 			display: block;
 		}
 
 		&-engagement-bar {
 			position: fixed;
-			bottom: 0;
+			bottom: var( --content-padding-bottom );
```

Comment:
The engagement bar wasn't aligning well with the scrollbar so moved it inside the card.